packages/@okta/vuepress-site/code/angular/okta_angular_auth_js/index.md

```diff
@@ -278,18 +333,26 @@ Finally, update your `@NgModule` to include your project components and routes i
   ],
   imports: [
     BrowserModule,
-    RouterModule.forRoot(appRoutes)
+    AppRoutingModule
   ],
   providers: [
     OktaAuthGuard,
     OktaAuthService,
   ],
-  bootstrap: [ AppComponent ]
+  bootstrap: [AppComponent]
 })
-
 export class AppModule { }
 ```
 
+Finally, build and start the app. In the terminal:
+
+```bash
+npm start
+```
+
+After a few moments, you should see a message:
```

Comment:
Lets re-word this to something like, `After the server is started, this message will appear in your terminal`